libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -210,6 +210,9 @@ class AC_AttitudeControl {
     // Return the body-to-NED target attitude used by the quadplane-specific attitude control input methods
     Quaternion get_attitude_target_quat() const { return _attitude_target; }
 
+    // Return the angular velocity of the target (setpoint) [rad/s] in the target attitude frame
+    const Vector3f & get_attitude_target_ang_vel() const { return _ang_vel_target;}
```

Comment:
nitpick: let's put the "&" immediately after the "Vector3f" (no space).

Reply:
fixed